src/Tizen.TV.UIControls.Forms/Handler/TVEditorHandler.cs

```diff
@@ -1,61 +1,50 @@
+ * limitations under the License.
+ */
 
-//using System;
-//using Microsoft.Maui.Controls;
-//using Microsoft.Maui.Handlers;
-//using TEntry = Tizen.UIExtensions.ElmSharp.Entry;
+using System;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Handlers;
+using TEditor = Tizen.UIExtensions.NUI.Editor;
 
-//namespace Tizen.TV.UIControls.Forms.Handler
-//{
-//    public class TVEditorHandler : EditorHandler
-//    {
-//        const string _doneKeyName = "Select";
-//        const string _cancelKeyName = "Cancel";
+namespace Tizen.TV.UIControls.Forms.Handler
+{
+    public class TVEditorHandler : EditorHandler
+    {
+        const string _doneKeyName = "Select";
 
-//        protected override void ConnectHandler(UIExtensions.ElmSharp.Entry platformView)
-//        {
-//            base.ConnectHandler(platformView);
+        protected override void ConnectHandler(TEditor platformView)
+        {
+            base.ConnectHandler(platformView);
 
-//            if (platformView != null)
-//            {
-//                //InputEvents.GetEventHandlers(VirtualView as BindableObject)?.Add(new RemoteKeyHandler(new Action<RemoteControlKeyEventArgs>((args) =>
-//                {
-//                    if (args.PlatformKeyName.Equals(_doneKeyName))
-//                    {
-//                        // TODO
-//                        //FocusSearch(true)?.SetFocus(true);
-//                        Device.BeginInvokeOnMainThread(() =>
-//                        {
-//                            VirtualView.Text = platformView.Text;
-//                            (VirtualView as IEditorController)?.SendCompleted();
-//                        });
-//                    }
-//                    else if (args.PlatformKeyName.Equals(_cancelKeyName))
-//                    {
-//                        platformView.HideInputPanel();
-//                    }
-//                }), RemoteControlKeyTypes.KeyDown));
-
-//                if (platformView is TEntry tentry)
-//                {
-//                    tentry.EntryLayoutFocused += OnFocused;
-//                    tentry.EntryLayoutUnfocused += OnUnfocused;
-//                }
-//            }
-//        }
-//    }
-//}
+            if (platformView != null)
+            {
+                InputEvents.GetEventHandlers(VirtualView as BindableObject)?.Add(new RemoteKeyHandler(new Action<RemoteControlKeyEventArgs>((args) =>
+                {
+                    if (args.PlatformKeyName.Equals(_doneKeyName))
+                    {
+                        // TODO
+                        //FocusSearch(true)?.SetFocus(true);
+                        ((BindableObject)VirtualView).Dispatcher.Dispatch(() =>
+                        {
+                            VirtualView.Text = platformView.Text;
+                            (VirtualView as IEditorController)?.SendCompleted();
+                        });
+                    }
+                }), RemoteControlKeyTypes.KeyDown));
+            }
```

Comment:
Is it ok not to handle focus event?

Reply:
It used to be handled in ElmSharp era because the entry is used to be wrapped by layout. Now with NUI implementation, only entry is used, so there is no need.